<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="toolTip">
      <string>Properties for current figure</string>
     </property>
     <property name="statusTip">
      <string>Properties for current figure</string>
     </property>
     <property name="title">
      <string>Properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,0,0,1,1">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLineEdit" name="QPlane_SetTitle_lineEdit">
        <property name="toolTip">
         <string>Set title of current figure</string>
        </property>
        <property name="statusTip">
         <string>Set title of current figure</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="QPlane_SetTitle_button">
        <property name="toolTip">
         <string>Set title of current figure</string>
        </property>
        <property name="statusTip">
         <string>Set title of current figure</string>
        </property>
        <property name="text">
         <string>Set Title</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="toolTip">
           <string>Set color scale axis of current figure</string>
          </property>
          <property name="statusTip">
           <string>Set color scale axis of current figure</string>
          </property>
          <property name="text">
           <string>C Axis max</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="toolTip">
           <string>Set color scale axis of current figure</string>
          </property>
          <property name="statusTip">
           <string>Set color scale axis of current figure</string>
          </property>
          <property name="text">
           <string>C Axis min</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="QPlane_CAxisMax_lineEdit">
          <property name="toolTip">
           <string>Set color scale axis of current figure</string>
          </property>
          <property name="statusTip">
           <string>Set color scale axis of current figure</string>
          </property>
          <property name="text">
           <string>0.02</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="QPlane_CAxisMin_lineEdit">
          <property name="toolTip">
           <string>Set color scale axis of current figure</string>
          </property>
          <property name="statusTip">
           <string>Set color scale axis of current figure</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="QPlane_setCAxis_button">
        <property name="toolTip">
         <string>Set color scale axis of current figure</string>
        </property>
        <property name="statusTip">
         <string>Set color scale axis of current figure</string>
        </property>
        <property name="text">
         <string>Set C Axis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="QPlane_Grid_checkBox">
        <property name="toolTip">
         <string>Show/hide grid on current figure</string>
        </property>
        <property name="statusTip">
         <string>Show/hide grid on current figure</string>
        </property>
        <property name="text">
         <string>Grid</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="toolTip">
         <string>Current Figure</string>
        </property>
        <property name="statusTip">
         <string>Current Figure</string>
        </property>
        <property name="text">
         <string>Current figure</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
        <item>
         <widget class="QComboBox" name="QPlane_figureList_comboBox">
          <property name="toolTip">
           <string>Current Figure</string>
          </property>
          <property name="statusTip">
           <string>Current Figure</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="QPlane_figureToCSV_pushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Export to csv</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="toolTip">
      <string>Properties for new plot</string>
     </property>
     <property name="statusTip">
      <string>Properties for new plot</string>
     </property>
     <property name="title">
      <string>Create QPlane</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="toolTip">
             <string>Bin size in 1/AA</string>
            </property>
            <property name="statusTip">
             <string>Bin size in 1/AA</string>
            </property>
            <property name="text">
             <string>Qy bin [1/AA]</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="toolTip">
             <string>Maximum energy in meV for integration plane</string>
            </property>
            <property name="statusTip">
             <string>Maximum energy in meV for integration plane</string>
            </property>
            <property name="text">
             <string>Energy Min [meV]</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="toolTip">
             <string>Minimum energy in meV for integration plane</string>
            </property>
            <property name="statusTip">
             <string>Minimum energy in meV for integration plane</string>
            </property>
            <property name="text">
             <string>Energy Max [meV]</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="QPlane_EMin_lineEdit">
            <property name="toolTip">
             <string>Maximum energy in meV for integration plane</string>
            </property>
            <property name="statusTip">
             <string>Maximum energy in meV for integration plane</string>
            </property>
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="QPlane_yBinTolerance_lineEdit">
            <property name="toolTip">
             <string>Bin size in 1/AA</string>
            </property>
            <property name="statusTip">
             <string>Bin size in 1/AA</string>
            </property>
            <property name="text">
             <string>0.03</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="QPlane_xBinTolerance_lineEdit">
            <property name="toolTip">
             <string>Bin size in 1/AA</string>
            </property>
            <property name="statusTip">
             <string>Bin size in 1/AA</string>
            </property>
            <property name="text">
             <string>0.03</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="toolTip">
             <string>Bin size in 1/AA</string>
            </property>
            <property name="statusTip">
             <string>Bin size in 1/AA</string>
            </property>
            <property name="text">
             <string>Qx bin [1/AA]</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="QPlane_EMax_lineEdit">
            <property name="toolTip">
             <string>Minimum energy in meV for integration plane</string>
            </property>
            <property name="statusTip">
             <string>Minimum energy in meV for integration plane</string>
            </property>
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="toolTip">
             <string>Units used when plotting</string>
            </property>
            <property name="statusTip">
             <string>Units used when plotting</string>
            </property>
            <property name="title">
             <string>Units</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QRadioButton" name="QPlane_SelectUnits_RLU_radioButton">
               <property name="text">
                <string>RLU</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="QPlane_SelectUnits_AA_radioButton">
               <property name="text">
                <string>AA-1</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="QPlane_LogScale_checkBox">
        <property name="toolTip">
         <string>Use log scale of intensity when plotting</string>
        </property>
        <property name="statusTip">
         <string>Use log scale of intensity when plotting</string>
        </property>
        <property name="text">
         <string>Log Scale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="QPlane_plot_button">
        <property name="toolTip">
         <string>Generate QPlane</string>
        </property>
        <property name="statusTip">
         <string>Generate QPlane</string>
        </property>
        <property name="text">
         <string>Plot Q plane</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>QPlane_SetTitle_lineEdit</tabstop>
  <tabstop>QPlane_SetTitle_button</tabstop>
  <tabstop>QPlane_CAxisMax_lineEdit</tabstop>
  <tabstop>QPlane_CAxisMin_lineEdit</tabstop>
  <tabstop>QPlane_setCAxis_button</tabstop>
  <tabstop>QPlane_Grid_checkBox</tabstop>
  <tabstop>QPlane_EMax_lineEdit</tabstop>
  <tabstop>QPlane_EMin_lineEdit</tabstop>
  <tabstop>QPlane_xBinTolerance_lineEdit</tabstop>
  <tabstop>QPlane_yBinTolerance_lineEdit</tabstop>
  <tabstop>QPlane_SelectUnits_RLU_radioButton</tabstop>
  <tabstop>QPlane_SelectUnits_AA_radioButton</tabstop>
  <tabstop>QPlane_LogScale_checkBox</tabstop>
  <tabstop>QPlane_plot_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
